Markdown lint rules (#1259)
GoogleChrome/web.dev: src/site/content/en/handbook/markup-code/index.md

```
````

> For code blocks that do not need syntax highlighting (e.g. HTTP headers) set

Reviewer: ```suggestion
For code blocks that don't need syntax highlighting (for example, HTTP headers), set
```

the language to `text`:

````markdown
```text
Set-Cookie: promo_shown=1; Max-Age=2600000; Secure
```
````

> For terminal commands set the language to `bash` and omit leading `$` signs:

Reviewer: ```suggestion
For terminal commands, set the language to `bash` and omit leading `$` signs:
```
